rgen/rgen-viewer/src/main.rs

```diff
@@ -268,11 +299,13 @@ impl Render {
     self.canvas.clear();
   }
 
-  pub fn present(&mut self) { self.canvas.present(); }
+  pub fn present(&mut self) {
+    self.canvas.present();
+  }
```

Comment:
you should change your formatter to the nightly branch

Reply:
could you explain that to me